docs/en/concepts-and-designs/lal.md

```diff
@@ -0,0 +1,191 @@
+# Log Analysis Language
+
+Log Analysis Language (LAL) in SkyWalking is essentially a Domain-Specific Language (DSL) to analyze logs. You can use
+LAL to parse, extract, filter, analyze, and save the logs, as well as collaborate the logs with traces, and metrics.
+
+Check the [`default.yaml`](../../../oap-server/server-bootstrap/src/main/resources/lal/default.yaml) for examples.
```

Comment:
Do we really have default LAL script?

Reply:
> Do we really have default LAL script?

I just removed that, the link will be pointed to the Envoy logs' config file after settled down